biojava-structure/src/main/java/org/biojava/nbio/structure/symmetry/core/SubunitContactGraph.java

```diff
@@ -45,17 +46,17 @@
 			.getLogger(SubunitContactGraph.class);
 
 	private static final double DISTANCE_CUTOFF = 8;
-	private static final int MIN_CONTACTS = 10;
+	private static final int MIN_CONTACTS = 5;
 
 	private SubunitContactGraph() {
 	}
 
-	public static UndirectedGraph<Integer, DefaultEdge> calculateGraph(
+	public static Graph<Integer, DefaultEdge> calculateGraph(
```

Comment:
It would be nice to do the contact calculation with the spatial hashing algorithm implemented elsewhere in biojava. The tutorial shows how to use it: https://github.com/biojava/biojava-tutorial/blob/master/structure/contact-map.md